### PR TITLE
Return an error if a user tries to update the priority of a job that …

### PR DESCRIPTION
**Pull request by AlexFalessi**
…has a bad value

### CODE REVIEW

**Review by nebilbenmabrouk — APPROVED** [review]
LGTM

File: rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java [hunk 1]
```diff
@@ -2435,11 +2437,25 @@ public void schedulerChangeJobPriorityByValue(String sessionId, String jobId, St
             throws RestException, JobAlreadyFinishedRestException {
         try {
             Scheduler s = checkAccess(sessionId, PATH_JOBS + jobId + "/priority/byvalue" + priorityValue);
-            s.changeJobPriority(jobId, JobPriority.findPriority(Integer.parseInt(priorityValue)));
-        } catch (SchedulerException e) {
-            throw RestException.wrapExceptionToRest(e);
+            JobPriority newPriority = Arrays.stream(JobPriority.values())
```
Comment (fviale) [inline rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java:2440]: a more elegant solution would be to add a findPriorityExact static method in JobPriority that throws an IllegalArgumentException if not found

Comment (AlexFalessi) [reply rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java:2440]: Ok i'll implement this.

**Comment by AlexFalessi** [comment]
@fviale I'll rollback the usage of a static map if you don't like it.

**Comment by fviale** [comment]
@AlexFalessi +1 to remove the map :-)

**Review by fviale — APPROVED** [review]

**Comment by AlexFalessi** [comment]
retest this please

**Pull request merged**